keycloak/README.md

```diff
@@ -2,12 +2,18 @@
 
```

Comment:
Please rewrite this REDME to link the upstream, instead of copying it. Also make sure to mention how to overwrite values from the upstream chart (ì.e. prepend `keycloak`).